### PR TITLE
[W4.11e][T09-4] XIAO DELONG

### PR DESCRIPTION
**Pull request by dlworldpeace**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,35 @@ private void assertAreUnique(Object... objects) {
...
+        assertContainsNull("abc", null, "abc", null, null);
+        assertContainsNull("abc", "", "ABC", null);
+        assertContainsNull(null, "", "abc", null, "a", "abc");
+
+        // all are null
+        assertContainsNull(null, null);
+    }
+
+    private void assertContainsNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:71]: Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,35 @@ private void assertAreUnique(Object... objects) {
...
+
+        // only one null
+        assertContainsNull(null, 1);
+        assertContainsNull("", null);
+        assertContainsNull("", null, "abc");
+
+        // some are null
+        assertContainsNull("abc", null, "abc", null, null);
+        assertContainsNull("abc", "", "ABC", null);
+        assertContainsNull(null, "", "abc", null, "a", "abc");
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:64]: Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.

**Review by eugenepeh — COMMENTED** [review]
Good job on adding both positive and negative test cases, as well as empty and null scenario. 

Some comments added. Please close the PR after reading the comments.
